src/main/java/com/scalyr/util/Util.java

```diff
@@ -0,0 +1,52 @@
+package com.scalyr.util;
+
+import com.scalyr.api.logs.EventAttributes;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class Util {
+  public static Integer stringToIntMemory(String maxBufferRam) {
+    if(maxBufferRam != null && !"".equals(maxBufferRam)) {
```

Comment:
Nit – missing space after "if", here and on line 12

Reply:
👍